.golangci.yml

```diff
@@ -57,17 +57,17 @@ linters:
     - errname
     - errorlint
     - exhaustive
-    - exportloopref
+    # - exportloopref
```

Comment:
Why commenting these out?

Reply:
@raulb it is no longer needed since 1.22.

I believe I did init this repo from the template before [this](https://github.com/ConduitIO/conduit-connector-template/commit/9227e64550fe551f316a2d1002193e9c0db79a91), let's add the `copyloopvar` instead.